sunpy/net/vso/table_response.py

```diff
@@ -91,17 +93,38 @@ def from_zeep_response(cls, response, *, client, _sort=True):
                         row["Wavetype"] = value['wavetype']
                         continue
                     for subkey, subvalue in value.items():
-                        key_template = f"{key.capitalize()} {subkey.capitalize()}"
                         if key == "time" and subvalue is not None:
                             key_template = f"{subkey.capitalize()} {key.capitalize()}"
-                            subvalue = parse_time(subvalue)
-                            # Change the display to the 'T'-less version
-                            subvalue.format = 'iso'
+                        else:
+                            key_template = f"{key.capitalize()} {subkey.capitalize()}"
                         row[key_template] = subvalue
             data.append(row)
 
-        # Reorder the columns to put the most useful ones first.
         data = cls(data, client=client)
+        # Parse times
+        for col in data.colnames:
+            if col.endswith('Time'):
+                try:
+                    # Try to use a vectorised call to parse_time
+                    data[col] = parse_time(data[col])
+                except Exception:
```

Comment:
Can we catch a more specific exception here?

Reply:
I think it's worth catching everything and falling back on the non-vectorised approach. If there's something very wrong (like an individual time can't be parsed, then an appropriate error will be raised in the except block.